hl7/_from_hl7.py

```diff
@@ -70,7 +70,13 @@ def _convert(hl7_content: str) -> Iterator[Tuple[str, str, str, str]]:
             for index2, field2 in enumerate(field.split("^"), start=1):
                 if not field2:
                     continue
-                description = _get_description(segment_name, index1, index2)
+                if index2 == 1:
+                   index2 = ""
+                   index1 = int(index1)
```

Comment:
This doesn't need a variable override. You can directly add `int(index1)` as an argument to the `_get_description` function.